features/unit/v2algodclient_paths.feature

```diff
@@ -69,3 +69,28 @@ Feature: Algod REST API v2 Paths
     Examples:
       | path                  | application-id |
       | /v2/applications/1234 | 1234           |
+
+  @unit.algod.ledger_refactoring
+  Scenario Outline: Account Information
+    When we make an Account Information call against account "<account>" with exclude "<exclude>"
+    Then expect the path used to be "<path>"
+    Examples:
+      | path                         | account | exclude |
+      | /v2/accounts/47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU             | 47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU     |         |
+      | /v2/accounts/47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU?exclude=all | 47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU     | all     |
```

Comment:
We should have a test that sets multiple values.
```suggestion
    Examples:
      | path                         | account | exclude |
      | /v2/accounts/47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU             | 47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU     |         |
      | /v2/accounts/47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU?exclude=all | 47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU     | all     |
      | /v2/accounts/47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU?exclude=assets,created-assets,apps-local-state,created-apps | 47YPQTIGQEO7T4Y4RWDYWEKV6RTR2UNBQXBABEEGM72ESWDQNCQ52OPASU     | assets,created-assets,apps-local-state,created-apps     |
```

Reply:
For algod, `exclude` can only take values `all` and `none`.

Reply:
great, thanks for clarifying that.